docs/concepts/multi-chain-proposals.md

```diff
@@ -0,0 +1,693 @@
+---
```

Comment:
[arbitrary comment location]

multi-chain proposals aren't showing up in the docs search (from http://localhost:3000/) -- do we need to manually index it, or would it be automatically picked up after deploy?

Reply:
I think this won't happen till after deploy